docs/src/Scenario2/Evaluation_Scenario_2.md

```diff
@@ -65,6 +77,10 @@ xmax, xmaxval = get_max_t(probne, sum(idart))
 @test isapprox(xmaxval, 0.6, atol = 0.01)
 ```
 
+#### Full Analysis of the Effect of Events
+
+TBD Paul
```

Comment:
@paulflang I gave you a section here. Fill with whatever data is required to reproduce the claims of the paragraph above (and make corrections if I said anything wrong)

Reply:
@ChrisRackauckas : see #42